src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 CHANGELOG
 =========
 
+6.x
```

Comment:
5.3 (or 5.4 after feature freeze)

Reply:
I would go with 5.4 but we need to add a php version id check somewhere 🧐

Reply:
Added the version check in the normalizer